risc0/cargo-risczero/docker/Dockerfile.release

```diff
@@ -0,0 +1,11 @@
+# To build run: docker build -f Dockerfile.release -t risczero/risc0-guest-builder:v0.17 .
+FROM ubuntu:20.04@sha256:3246518d9735254519e1b2ff35f95686e4a5011c90c85344c1f38df7bae9dd37
```

Comment:
small nit 20.04 will EOL in 2024, vs 22.04 which is is EOL (base support tier)  in 2026

Should we jump to 22.04 right now to extend the lifecycle of these images?